site/docs/webrtc/voice-iw.md

```diff
@@ -0,0 +1,191 @@
+---
+id: voice-iw
+title: Voice Interworking
+slug: /webrtc/voice-iw
+description: A description of available options for interworking with Bandwidth Programmable Voice
+keywords:
+  - bandwidth
+  - webrtc
+  - video
+image: ../../static/img/bandwidth-logo.png
+---
+
+## Connecting WebRTC to Programmable Voice
+
+Bandwidth has a demonstrated history of excellence in IP voice technologies, including comprehensive CPaaS Programmable Voice and Messaging capabilities. The introduction of WebRTC capabilities provides the opportunity to combine the strength and diversity of our Programmable Voice capabilities with our highly controllable WebRTC capabilities to yield a rich and flexible platform that allows our customers to extend their business with a broad range of communication capabilities.
+
+One of the steps we have taken to enable this capability suite is to ensure that our Programmable Voice (PV) and WebRTC platforms and APIs combine their respective strengths when integrated.
```

Comment:
I think you can remove 'and APIs'

Reply:
I chose to remove platforms, because the APIs are what the users consume.